app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -546,11 +517,35 @@ open class DefaultToolbarMenu(
     val deleteDataOnQuit = BrowserMenuImageText(
         label = context.getString(R.string.delete_browsing_data_on_quit_action),
         imageResource = R.drawable.ic_exit,
-        iconTintColorResource = primaryTextColor
+        iconTintColorResource = primaryTextColor()
     ) {
         onItemTapped.invoke(ToolbarMenu.Item.Quit)
     }
 
+    val syncedTabsItem = BrowserMenuImageText(
+        context.getString(R.string.synced_tabs),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor()
+    ) {
+        onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)
+    }
+
+    val syncItemTitle =
+        if (accountManager.accountProfile()?.email != null) {
+            signedInToFxA = true
+            accountManager.accountProfile()?.email!!
```

Comment:
Do we need the !! when we check that it's not null in the if statement?

Reply:
I'm gonna take a look at other places in the code and redo this part